pillar/heroku/bootcamps.sls

```diff
@@ -146,6 +148,7 @@ heroku:
     NOVOED_SAML_DEBUG: {{ env_data.NOVOED_SAML_DEBUG }}
     NOVOED_SAML_KEY: __vault__::secret-{{ business_unit }}/{{ env_data.vault_env_path }}/novoed>data>saml_key
     NOVOED_SAML_CERT: __vault__::secret-{{ business_unit }}/{{ env_data.vault_env_path }}/novoed>data>saml_cert
+    NOVOED_SAML_LOGIN_URL: {{ env_data.NOVOED_SAML_LOGIN_URL }}
```

Comment:
That would fail for CI as it was be undefined. Don't think there would be harm in hooking up CI to staging as well so maybe just add the url to ci as well.

Reply:
Done 👍